frontend/src/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import i18n from '../common/config/i18n';
+import Button from './Button';
+import Modal from './Modal';
+
+interface ConfirmationModalProps {
```

Comment:
More often than not this confirms and option to delete something and in order to avoid keeping a different state to know what we delete this should also contain the id of the element we perform de "confirm" action on and also send it back.

The complete interface should be as follows:
interface ConfirmationModalProps {
  id?: string;
  title: string;
  description: string;
  confirmBtnLabel: string;
  confirmBtnClassName: string;
  onClose: () => void;
  onConfirm: (id?: string) => void;
}